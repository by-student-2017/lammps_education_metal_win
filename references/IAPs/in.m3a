# Anharmonic potential for TITEP+M3A in c-Si. 

neighbor 0.0 bin
boundary p p p  
units metal

atom_style angle

atom_modify map array  
read_data data.m3a

bond_style morse # Need to overlay a separate bond style describing TITEP interactions.
angle_style class2 # Allows definition of anharmonic angle terms.

pair_coeff * *

# Bond Coeffs
# All other bonds must be described by a TITEP.
bond_coeff 8 morse 0.358100 0.585600 3.869288 
bond_coeff 9 morse 2.177700 1.324500 2.369446 
bond_coeff 14 morse 0.358100 0.585600 3.869288 
bond_coeff 15 morse 0.358100 0.585600 3.869288 
bond_coeff 16 morse 0.358100 0.585600 3.869288 
bond_coeff 17 morse 2.177700 1.324500 2.369446 
bond_coeff 18 morse 0.358100 0.585600 3.869288 
bond_coeff 19 morse 0.358100 0.585600 3.869288 
bond_coeff 55 morse 2.177700 1.324500 2.369446 
bond_coeff 56 morse 2.177700 1.324500 2.369446 
bond_coeff 62 morse 0.358100 0.585600 3.869288 
bond_coeff 65 morse 0.358100 0.585600 3.869288 
bond_coeff 73 morse 0.358100 0.585600 3.869288 
bond_coeff 81 morse 0.358100 0.585600 3.869288 
bond_coeff 84 morse 0.358100 0.585600 3.869288 
bond_coeff 89 morse 0.358100 0.585600 3.869288 

#Angle Coeffs
angle_coeff 1 35.264348 -0.351900 0.125200 -2.460200
angle_coeff 1 bb 0.0 0.0 0.0
angle_coeff 1 ba 0.0 0.0 0.0 0.0

angle_coeff 2 90.000000 -2.387100 -1.238800 -0.187500
angle_coeff 2 bb 0.0 0.0 0.0
angle_coeff 2 ba 0.0 0.0 0.0 0.0

angle_coeff 3 144.735652 0.014700 0.000000 -4.999400
angle_coeff 3 bb 0.0 0.0 0.0
angle_coeff 3 ba 0.0 0.0 0.0 0.0

angle_coeff 4 60.000000 0.000000 0.163800 -0.298300
angle_coeff 4 bb 0.0 0.0 0.0
angle_coeff 4 ba 0.0 0.0 0.0 0.0

angle_coeff 5 90.000000 0.0 0.0 0.0
angle_coeff 5 bb 0.0 0.0 0.0
angle_coeff 5 ba 0.0 0.0 0.0 0.0

angle_coeff 6 120.000000 0.0 0.0 0.0
angle_coeff 6 bb 0.0 0.0 0.0
angle_coeff 6 ba 0.0 0.0 0.0 0.0

angle_coeff 7 109.471200 0.506300 0.152200 3.501000
angle_coeff 7 bb 0.0 0.0 0.0
angle_coeff 7 ba 0.0 0.0 0.0 0.0

angle_coeff 8 180.000000 0.0 0.0 0.0
angle_coeff 8 bb 0.0 0.0 0.0
angle_coeff 8 ba 0.0 0.0 0.0 0.0
