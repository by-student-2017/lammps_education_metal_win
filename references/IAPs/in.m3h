# Anharmonic potential for TITEP+M3H in c-Si. 

neighbor 0.0 bin
boundary p p p  
units metal

atom_style angle

atom_modify map array  
read_data data.m3h

bond_style morse # Need to overlay a separate bond style describing TITEP interactions.
angle_style harmonic

pair_coeff * *

# Bond Coeffs
# All other bonds must be described by a TITEP.
bond_coeff 8 morse 7.304900 0.055000 3.869288
bond_coeff 9 morse 2.351000 1.291100 2.369446
bond_coeff 14 morse 7.304900 0.055000 3.869288 
bond_coeff 15 morse 7.304900 0.055000 3.869288 
bond_coeff 16 morse 7.304900 0.055000 3.869288 
bond_coeff 17 morse 2.351000 1.291100  2.369446 
bond_coeff 18 morse 7.304900 0.055000 3.869288
bond_coeff 19 morse 7.304900 0.055000 3.869288
bond_coeff 55 morse 2.351000 1.291100  2.369446
bond_coeff 56 morse 2.351000 1.291100  2.369446
bond_coeff 65 morse 7.304900 0.055000 3.869288
bond_coeff 73 morse 7.304900 0.055000 3.869288
bond_coeff 81 morse 7.304900 0.055000 3.869288
bond_coeff 84 morse 7.304900 0.055000 3.869288

#Angle Coeffs
angle_coeff 1 -0.628600 35.264348
angle_coeff 2 0.244200 90.000000
angle_coeff 3 0.019600 144.735652
angle_coeff 4 0.000000 60.000000
angle_coeff 5 0.000000 90.000000
angle_coeff 6 -0.000100 120.000000
angle_coeff 7 0.935800 109.471200
angle_coeff 8 0.000000 180.000000
