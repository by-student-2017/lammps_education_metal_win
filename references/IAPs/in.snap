# Anharmonic potential for TITEP+M3A in c-Si. 

neighbor 0.0 bin
boundary p p p  
units metal

atom_style angle

atom_modify map array  
read_data data.gan

pair_style snap # Need to overlay a separate pair style describing TITEP interactions.
pair_coeff * * snap GaN.snapcoeff Ga N GaN.snapparam Ga N
